keychain/btcwallet.go

```diff
@@ -458,8 +458,8 @@ func (b *BtcWalletKeyRing) SignMessageCompact(keyLoc KeyLocator,
 //
 // NOTE: This is part of the keychain.MessageSignerRing interface.
 func (b *BtcWalletKeyRing) SignMessageSchnorr(keyLoc KeyLocator,
-	msg []byte, doubleHash bool, taprootTweak []byte) (*schnorr.Signature,
-	error) {
+	msg []byte, doubleHash bool, taprootTweak []byte, tag []byte,
+) (*schnorr.Signature, error) {
```

Comment:
nit: I know we've started using this formatting in some places, but I personally really don't like it :sweat_smile: So I always try to argue not to use it if there is an alternative.
Here it would be pretty easy by just breaking before the `tag` parameter to avoid this non-indented part of the function header. Same in other places throughout this commit.